.github/workflows/verify-formatting.yaml

```diff
@@ -0,0 +1,37 @@
+name: 'Verify files formatting'
+
+on:
+  push:
+    paths:
+      - '**/*.md'
+      - '**/*.yml'
+      - '**/*.yaml'
+      - '.prettierrc'
+      - '.prettierignore'
+      - '!.github/ISSUE_TEMPLATE/**/*'
+      - '!.github/PULL_REQUEST_TEMPLATE.md'
```

Comment:
Github doesn't use the standard flavour of Markdown but adds some custom extensions to it, I thought of excluding those to avoid Prettier mishandling them. 
To be fair it should work most of the times so we could avoid excluding them.

Reply:
As you think best. I was only asking so that I could more fully understand the system.

I do think GFM support would be an important thing generally, even if not so essential in this repository where almost all of the Markdown content is not rendered as GFM. I do notice that they mention GFM right on the Prettier homepage, so at least it seems like they intend to support it, as opposed to Python Markdown, where they actively oppose support for any variation from the original specification.

Reply:
I've removed them by the way, I didn't notice Prettier supported GitHub-Flavored Markdown.